user/tests/libraries/unit-test/flashee-eeprom-impl.h

```diff
@@ -1286,80 +1286,4 @@ class SinglePageWear : public ForwardingFlashDevice {
 };
 
 
-#ifdef FLASHEE_EEPROM
```

Comment:
Should be kept and instead be conditional for `HAS_SERIAL_FLASH`, otherwise P1 builds fail.

Reply:
fixed

Reply:
I don't know why I deleted this. This is also what broke the build. I have restored it, under `HAS_SERIAL_FLASH`.